widget/embedded/src/components/WalletModal/WalletModal.tsx

```diff
@@ -0,0 +1,74 @@
+import type { ModalContentProps, PropTypes } from './WalletModal.types';
+
+import { i18n } from '@lingui/core';
+import {
+  BottomLogo,
+  Divider,
+  Image,
+  MessageBox,
+  Modal,
+  WalletState,
+} from '@rango-dev/ui';
+import React from 'react';
+
+import {
+  LogoContainer,
+  Spinner,
+  WalletImageContainer,
+} from './WalletModal.styles';
+
+export function WalletModal(props: PropTypes) {
+  const { open, onClose, ...otherProps } = props;
+
+  return (
+    <Modal
+      open={open}
+      onClose={onClose}
+      container={document.getElementById('swap-box') || document.body}>
+      <ModalContent {...otherProps} />
+      <Divider direction="vertical" size={32} />
+      <BottomLogo />
+    </Modal>
+  );
+}
+
+function ModalContent(props: ModalContentProps) {
```

Comment:
Let's keep a single component for each file. please make a new component `widget/embedded/src/components/WalletModal/ModalContent.tsx`